src/react/features/connector/mockResponses/GET_ROOM.json

```diff
@@ -4,6 +4,8 @@
     "room1": {
       "id": "room1",
       "text": "Room 1",
+      "latitude": 60.17037863792087,
+      "longitude": 24.941368103027347,
```

Comment:
What is the use case for putting room diory on a map?

For me it doesn't make too much sense...